applications/nrf_desktop/README.rst

```diff
@@ -863,11 +863,37 @@ Selecting a build type from command line
 
 .. note::
    If nRF Desktop is built with `Fast Pair`_ support, you must provide your own Fast Pair Model ID and Anti Spoofing private key as CMake options.
+   You can use either your own provisioning data or the provisioning data obtained by Nordic Semiconductor for development purposes.
+   The following debug devices are meant to be used with the nRF Desktop and have been registered:
+
+   * NCS keyboard - The Fast Pair Provider meant to be used with keyboards:
+
+      * Device name: NCS keyboard
+      * Model ID: ``0x52FF02``
+      * Anti-spoofing key (base64, uncompressed): ``8E8ulwhSIp/skZeg27xmWv2SxRxTOagypHrf2OdrhGY=``
+      * Type: Input device
+      * Additional features: Data only connection, Support personalized name, Ringing device unsupported
+
+   * NCS gaming mouse - Fast Pair Provider meant to be used with gaming mice:
+
+      * Device name: NCS gaming mouse
+      * Model ID: ``0x8E717D``
+      * Anti-spoofing key (base64, uncompressed): ``dZxFzP7X9CcfLPC0apyRkmgsh3n2EbWo9NFNXfVuxAM=``
+      * Type: Input device
+      * Additional features: Data only connection, Support personalized name, Ringing device unsupported
+
    See :ref:`ug_bt_fast_pair_provisioning` documentation for the following information:
 
    * Registering a Fast Pair Provider
    * Provisioning a Fast Pair Provider in |NCS|
 
+.. important::
```

Comment:
This important admonitions is used in two places. We could use the ".. include::" sytnax for this.
We could use the following labels before line no: 890 and after line no 896 respectively :

.. nrf_desktop_fastpair_important_start
.. nrf_desktop_fastpair_important_end

and in the sample documentation, where this admonitions is called, add the following "include" syntax.

.. include:: /applications/nrf_desktop/README.rst
   :start-after: nrf_desktop_fastpair_important_start
   :end-before: nrf_desktop_fastpair_important_end

Reply:
This could be fixed along with the comment : https://github.com/nrfconnect/sdk-nrf/pull/11114#discussion_r1192461168, in that PR.

Reply:
Done. It needed a little reword because one was written in singular form and one was written in plural.